tests/local.at

```diff
@@ -733,10 +733,35 @@ void
   ++global_nerrs;
   ++*nerrs;]])[
   std::cerr << ]AT_LOCATION_IF([l << ": " << ])[m << '\n';
-}]])
+}]AT_ERROR_CUSTOM_IF([[
+void
+]AT_NAMESPACE[::parser::yyreport_syntax_error (const Context& ctx) const
+{
+  /* Arguments of yyformat: reported tokens (one for the "unexpected",
+     one per "expected"). */
+  int arg[yyntokens_];
+  int n = ctx.yysyntax_error_arguments (arg, yyntokens_);]AT_PARAM_IF([m4_bpatsubst(m4_defn([AT_PARSE_PARAMS]),
```

Comment:
Gee, I'm sorry you had to deal with the testsuite idiosyncrasies, but also very happy you did :)

Reply:
Fortunately, I had the C tests as a blueprint :)